packages/ui/components/profile/ProfileMemberCard.tsx

```diff
@@ -1,5 +1,153 @@
-export interface ProfileMemberCardProps {}
+import clsx from 'clsx'
+import { useTranslation } from 'react-i18next'
 
-export const ProfileMemberCard = ({}: ProfileMemberCardProps) => {
-  return <p>placeholder</p>
+import { Button } from '../Button'
+import { MembershipPill } from './MembershipPill'
+import { ProfileCardWrapper } from './ProfileCardWrapper'
+
+// Represents a tranch of tokens that is currently unstaking and will become
+// available at some well-known later date.
+export interface UnstakingTokensTranch {
+  tokenSymbol: string
+  unstakingTokens: number
+  available: Date
+}
+
+export interface ProfileMemberCardProps {
+  loadingClaiming?: boolean
+  loadingManaging?: boolean
+  votingPower: number
+  daoName: string
+  walletName: string
+  profileImgUrl: string
+  tokenSymbol: string
+  stakedTokens: number
+  tokenDecimals: number
+  unstakingTokensTranches: UnstakingTokensTranch[]
+  unstakedTokens: number
+  openProposals?: boolean
+  established: Date
+}
+
+export const ProfileMemberCard = ({
+  loadingClaiming,
+  loadingManaging,
+  votingPower,
+  daoName,
+  walletName,
+  profileImgUrl,
+  tokenSymbol,
+  stakedTokens,
+  tokenDecimals,
+  unstakingTokensTranches,
+  unstakedTokens,
+  openProposals,
+  established,
+}: ProfileMemberCardProps) => {
+  const { t } = useTranslation()
+
+  return (
+    <ProfileCardWrapper
+      childContainerClassName="p-0 border-t-0"
+      established={established}
+      imgUrl={profileImgUrl}
+      underHeaderComponent={<MembershipPill daoName={daoName} isMember />}
+      walletName={walletName}
+    >
+      <div className="py-6 px-6 border-t border-t-border-primary">
```

Comment:
```suggestion
      <div className="p-6 pb-4 border-t border-t-border-primary">
```

Subtle design tweak to match Figma.